### PR TITLE
Issue15, Add stats block support and infinity icon

### LINKED ISSUE #15: Stats
stats

Can leverage columns code but should be its own block

### PR DESCRIPTION
**Pull request by jfait**
Fix #15
https://github.com/hlxsites/caesars/issues/15

🔗 Test URLs:
- Before: https://main--caesars--hlxsites.hlx.page/caesars-palace/drafts/jfait/stats
- After: https://15-block-stats--caesars--hlxsites.hlx.page/caesars-palace/drafts/jfait/stats

📝 Description:
  
Target : Stats seen on https://www.caesars.com/caesars-palace
image

Authoring
image

### CODE REVIEW

File: caesars-palace/blocks/stats/stats.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
+/*
+Stats Block (2->3->5 columns per row across key media breakpoints)
+*/
+
+div.stats-container {
```
Comment (tmathern) [inline caesars-palace/blocks/stats/stats.css:5]: I think it's better to scope the CSS to the block, like [here](https://github.com/hlxsites/caesars/blob/main/caesars-palace/blocks/highlight-card/highlight-card.css#L5)

File: caesars-palace/blocks/stats/stats.css [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    padding: 0;
+}
+
+/* Stats Grid */
+div.stats-container .stats > div{
```
Comment (tmathern) [inline caesars-palace/blocks/stats/stats.css:10]:
```suggestion
div.stats-container .stats > div {
```

File: caesars-palace/blocks/stats/stats.css [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    display: grid;
+    grid-template-columns: repeat(2,1fr);
+    row-gap: 16px;
+    column-gap: 16px;
+}
+
+/* Stats Cell */
+div.stats-container .stats > div > div{
```
Comment (tmathern) [inline caesars-palace/blocks/stats/stats.css:18]:
```suggestion
div.stats-container .stats > div > div {
```

File: caesars-palace/blocks/stats/stats.css (hunk 1 above)
Comment (jfait) [reply caesars-palace/blocks/stats/stats.css:5]: Not sure i understand.

image

**Review by amol-anand — APPROVED** [review]

File: caesars-palace/styles/styles.css [hunk 4]
```diff
@@ -30,7 +30,8 @@
   /* fonts */
   --body-font-family: 'Open Sans',sans-serif;
   --heading-font-family: 'Josefin Sans',sans-serif;
-   --fixed-font-family: "Roboto Mono", menlo, consolas, "Liberation Mono", monospace;
+  --fixed-font-family: "Roboto Mono", menlo, consolas, "Liberation Mono", monospace;
+  --montserrat-font-family: 'Montserrat',sans-serif;
```
Comment (arumsey) [inline caesars-palace/styles/styles.css:34]: CSS var names should abstract away what they are defining. In this case calling the var `montserrat-font-family` isn't very useful. I think a better way would have been to override the `--heading-font-family` within your component that wanted to use a different font.

**Pull request merged**